src/app/proxy/proxyUtils.js

```diff
@@ -29,6 +29,8 @@ const apiInterceptor = new ApiInterceptor(keyCloakConfig, cacheConfig)
 const decorateRequestHeaders = function (upstreamUrl = "") {
   return function (proxyReqOpts, srcReq) {
     var channel = _.get(srcReq, 'session.rootOrghashTagId') || _.get(srcReq, 'headers.X-Channel-Id') || envHelper.DEFAULT_CHANNEL
+    var sessionId = _.get(srcReq, 'session.id') || _.get(srcReq, 'sessionID');
```

Comment:
This should be
var sessionId = _.get(srcReq, 'session.id') || _.get(srcReq, 'headers.X-Session-Id');